cmd/juju/action/show.go

```diff
@@ -102,10 +102,21 @@ func (c *showCommand) Run(ctx *cmd.Context) error {
 			if !ok {
 				continue
 			}
-			args[argName] = actionArg{
-				Type:        infoMap["type"],
-				Description: infoMap["description"],
+
+			if infoMap["default"] != nil {
+				args[argName] = actionArg{
+					Type:        infoMap["type"],
+					Description: infoMap["description"],
+					Default:     infoMap["default"],
```

Comment:
setting an interface{} to 'nil' is the same as not setting it, so I think you can just have
`args[argName] = actionArg{Type=,Description=,Default=}`

without doing any other checking.

Reply:
Much nicer, fixed. Thanks!